tests/test_videos.py

```diff
@@ -95,46 +94,6 @@ def make_video_file(video_file_file, language='en'):
     return VideoFile(video_file_file.name, language=language)
 
 
-@pytest.fixture
-def srt_subtitles_file():
-    if not os.path.exists("tests/testcontent/original.srt"):
-        with open("tests/testcontent/original.srt", 'wb') as f:
-            resp = requests.get(
-                'https://raw.githubusercontent.com/learningequality/'
-                'pressurecooker/master/tests/files/original.srt',
```

Comment:
This was 404 after new release of pressurecooker, so instead I manually added the file to the test files subs dir.